cypress/e2e/tests/migration/migration-waves/sort.test.ts

```diff
@@ -93,6 +99,71 @@ describe(["@tier2"], "Migration Waves sort validations", function () {
     });
+    it("Sort Manage applications table", function () {
+        const applicationsList: Assessment[] = [];
+
+        const stakeholdersList = createMultipleStakeholders(3);
+        const businessServicesList = createMultipleBusinessServices(3);
+        for (let i = 0; i < 3; i++) {
+            let appdata = {
+                name: data.getRandomWord(4),
+                owner: stakeholdersList[i].name,
+                business: businessServicesList[i].name,
+            };
+            const application = new Assessment(appdata);
+            application.create();
+            applicationsList.push(application);
+        }
+        const { start: startDate, end: endDate } = generateRandomDateRange();
+        const migrationWave = new MigrationWave(data.getRandomWord(4), startDate, endDate);
+        migrationWave.create();
+        migrationWave.openManageApplications();
+        const unsortedappList = getTableColumnData("Application Name");
+        const unsortedbusinessList = getTableColumnData("Business service");
+        const unsortedOwnerList = getTableColumnData("Owner");
+        //sort asc and desc for 3 columns
+        clickOnSortButton(
+            "Business service",
+            SortType.ascending,
+            MigrationWaveView.manageApplicationsTable
+        );
+        const afterAscSortBusinessList = getTableColumnData("Business service");
+        verifySortAsc(afterAscSortBusinessList, unsortedbusinessList);
+        clickOnSortButton(
+            "Application Name",
+            SortType.ascending,
+            MigrationWaveView.manageApplicationsTable
+        );
+        const afterAscSortappList = getTableColumnData("Application Name");
+        verifySortAsc(afterAscSortappList, unsortedappList);
+
+        clickOnSortButton("Owner", SortType.ascending, MigrationWaveView.manageApplicationsTable);
+        const afterAscSortOwnerList = getTableColumnData("Owner");
+        verifySortAsc(afterAscSortOwnerList, unsortedOwnerList);
+        clickOnSortButton(
+            "Business service",
+            SortType.ascending,
+            MigrationWaveView.manageApplicationsTable
+        );
+        const afterDescSortBusinessList = getTableColumnData("Business service");
+        verifyDateSortDesc(afterDescSortBusinessList, unsortedbusinessList);
+        clickOnSortButton(
+            "Application Name",
+            SortType.ascending,
+            MigrationWaveView.manageApplicationsTable
+        );
+        const afterDescSortappList = getTableColumnData("Application Name");
+        verifyDateSortDesc(afterDescSortappList, unsortedappList);
+
+        clickOnSortButton("Owner", SortType.ascending, MigrationWaveView.manageApplicationsTable);
+        const afterDescSortOwnerList = getTableColumnData("Owner");
+        verifyDateSortDesc(afterDescSortOwnerList, unsortedOwnerList);
+        cancelForm();
+        deleteByList(applicationsList);
```

Comment:
These have to part of clean up code in after method.
All deleteByList

Reply:
@sshveta they are not a global variable since they wont be used in other test cases, should i make them global?